public/js/template.js

```diff
@@ -0,0 +1,52 @@
+import homeView from '../../views/home.hbs';
+import { initEventListeners } from './filter_listeners';
+import { initCalendarEventListener } from './main.js';
+
+let eventsState = [];
+
+export function renderStuff(newFilter) {
+  let filteredEvents = eventsState.filter(event => event[newFilter.lang]);
+  let newDateTo;
+  let newDateFrom;
+  if (newFilter.date_from) {
+    newDateTo = new Date('2018-11-2');
+    newDateFrom = new Date(newFilter.date_from);
+    filteredEvents = filteredEvents.filter(
+      event =>
+        new Date(event.startTime) >= newDateFrom &&
+        new Date(event.startTime) <= newDateTo
+    );
+  } else if (newFilter.categories && !newFilter.date_from) {
+    newDateFrom = new Date();
+    filteredEvents = filteredEvents.filter(
+      event => new Date(event.startTime) > newDateFrom
+    );
+  } else {
+    newDateFrom = new Date();
+    filteredEvents = filteredEvents.filter(
+      event => new Date(event.startTime) > newDateFrom
+    );
+  }
+
+  if (newFilter.categories) {
+    filteredEvents = filteredEvents.filter(event =>
```

Comment:
You could probably refactor this section so there's just one filter that applies all the options. It would stop this code from looking so janky and also remove the need for multiple loops. If there are a bunch of if statements there's usually something not right.